client/src/node/main.ts

```diff
@@ -602,4 +602,19 @@ export class SettingMonitor {
 			void this._client.stop();
 		}
 	}
-}
\ No newline at end of file
+}
+
+function handleChildProcessStartError(process: ChildProcess, message: string) {
+	if (process == null) {
+		return Promise.reject<MessageTransports>(message);
+	}
+
+	return new Promise<MessageTransports>((_, reject) => {
+		process.on("error", (err) => {
```

Comment:
One minor thing: we use single quotes for constant string values (e.g. 'error')

Reply:
Done!